### DISCUSSION TITLE
How to achieve an glow effect

### DISCUSSION THREAD

**Discussion by JonaNorman** [google/filament#6771] [Q&A]
I'm an Android developer developing an AR app. There is a requirement for the model to have an glow effect, but I found that the glb models exported through C4D's Octane rendering engine do not have the glow effect.
Later on, I discovered that glb models do not support the glow effect and it needs to be achieved through post-processing effects. I found that Babylon.js can achieve the glow effect, and here's the demo address: https://playground.babylonjs.com/#7QCYPB. 
I am curious about how to achieve this with Filament, and I look forward to your reply.
<img width="413" alt="glow effect" src="https://user-images.githubusercontent.com/4536178/234450443-1b4a7336-0b94-4eac-8b0d-a24be042508c.png">

**Comment by romainguy** [comment]
This can be achieved by giving the gltf an emissive color and turning on the bloom post processing effect on the Filament View.

**Reply by JonaNorman** [reply]
Does the bloom effect only affect the area where the emissive color is set?

**Reply by romainguy** [reply]
No, and no real time rendering engine does this afaik (it's a screen space effect). But areas that are not strongly emissive will not appear to bloom.

**Reply by romainguy** [reply]
See examples of bloom here https://github.com/google/filament/releases/tag/v1.8.0 (tone mapping section)

**Reply by romainguy** [reply]
And here: https://user-images.githubusercontent.com/869684/77979007-54c58080-72b8-11ea-8042-d14e63602fc2.png

**Reply by JonaNorman** [reply]
Thanks a lot, I will give it a try.
